modules/installation/proc-config-dynamic-plugins-rhdh-operator.adoc

```diff
@@ -22,49 +36,32 @@ data:
         pluginConfig: {}
 ----
 
-[NOTE]
-====
-If the `pluginConfig` field references environment variables, you must define the variables in your `secrets-rhdh` secret.
-====
-
-.Procedure
-. From the {ocp-short} web console, select the *ConfigMaps* tab.
-. Click *Create ConfigMap*.
-. From the *Create ConfigMap* page, select the *YAML view* option in *Configure via* and edit the file, if needed.
 . Click *Create*.
 . Go to the *Topology* view.
 . Click on the overflow menu for the {product} instance that you want to use and select *Edit Backstage* to load the YAML view of the {product} instance.
 +
 image::rhdh/operator-install-2.png[]
 
-. Add the `spec.application.dynamicPluginsConfigMapName` field to the Custom Resource. For example:
+. Add the `dynamicPluginsConfigMapName` field to your `Backstage` CR. For example:
 +
 [source,yaml]
 ----
+apiVersion: rhdh.redhat.com/v1alpha1
+kind: Backstage
+metadata:
+  name: my-rhdh
 spec:
   application:
-    appConfig:
-      mountPath: /opt/app-root/src
-      configMaps:
-        - name: app-config-rhdh
+# ...
     dynamicPluginsConfigMapName: dynamic-plugins-rhdh
-    extraEnvs:
-      secrets:
-        - name: secrets-rhdh
-    extraFiles:
-      mountPath: /opt/app-root/src
-    replicas: 1
-    route:
-      enabled: true
-  database:
-    enableLocalDb: true
+# ...
 ----
 . Click *Save*.
 . Navigate back to the *Topology* view and wait for the {product} pod to start.
 . Click the *Open URL* icon to start using the {product} platform with the new configuration changes.
 
 .Verification
 
-* To check that the dynamic plugins configuration has been loaded, append the following to your {product} root URL: `/api/dynamic-plugins-info/loaded-plugins` and check the list of plugins.
-
-image::rhdh/operator-install-3.png[]
\ No newline at end of file
+* Ensure that the dynamic plugins configuration has been loaded, by appending `/api/dynamic-plugins-info/loaded-plugins` to your {product} root URL and then checking the list of plugins.
++
+image::rhdh/operator-install-3.png[]
```

Comment:
This image is a bit small, making it hardly readable. Would it make sense to get rid of the image and display a sample text output instead?

Reply:
I thought about doing exactly this, but left it alone because it wasn't part of the scope of the original Jira I was working on 😄 If you can provide maybe an example command / output here, I would love to include that instead 🙏🏻

Reply:
Sure - here is an example of output:
```json
[
  {
    "name": "backstage-plugin-catalog-backend-module-github-dynamic",
    "version": "0.5.2",
    "platform": "node",
    "role": "backend-plugin-module"
  },
  {
    "name": "backstage-plugin-techdocs",
    "version": "1.10.0",
    "role": "frontend-plugin",
    "platform": "web"
  },
  {
    "name": "backstage-plugin-techdocs-backend-dynamic",
    "version": "1.9.5",
    "platform": "node",
    "role": "backend-plugin"
  },
]
```